include/TPP/Passes.td

```diff
@@ -256,6 +247,16 @@ def RewriteBatchMatmulToMatmul : Pass<"rewrite-batch-matmul-to-matmul",
   let dependentDialects = ["scf::SCFDialect", "linalg::LinalgDialect"];
 }
 
+def CombineXsmmOpPass : Pass<"combine-xsmm-op-optimization", "func::FuncOp"> {
+  let summary = "Fuse gemm-relu-bias into a bigger xsmm op";
```

Comment:
I'd be more generic here. We want to add all sorts of XSMM fusions in this pass. For now we only have those.